src/Navigation/NavRouterLink.js

```diff
@@ -1,17 +1,31 @@
 import React, { PropTypes } from 'react';
 import { Link } from 'react-router';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
```

Comment:
No, it's not being used! Sorry about that. It was leftover from when we were going to make the nested menu with a state change. I meant to clean everything up but I guessed I missed that. I'll go ahead and remove it. Thanks for catching that, David!